### PR TITLE
Todo changes for coverlet inproc datacollector

### PR DESCRIPTION
**Pull request by vagisha-nidhi**
## Description
Todo changes for coverlet inproc datacollector

## Related PR
https://github.com/microsoft/vstest/pull/1969

### CODE REVIEW

File: src/vstest.console/Processors/CollectArgumentProcessor.cs [hunk 1]
```diff
@@ -229,17 +229,15 @@ internal static class CoverletConstants
...
             /// </summary>
-            public const string CoverletDataCollectorAssemblyQualifiedName = "Microsoft.TestPlatform.Extensions.CoverletCoverageDataCollector.CoverletCoverageDataCollector, Microsoft.TestPlatform.Extensions.CoverletCoverageDataCollector, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a";
+            public const string CoverletDataCollectorAssemblyQualifiedName = "Coverlet.Collector.DataCollection.CoverletInProcDataCollector, coverlet.collector, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null";
 
-            /// TODO : Finalize codebase
             /// <summary>
             /// Coverlet inproc data collector codebase
             /// </summary>
-            public const string CoverletDataCollectorCodebase = "coverletinprocdatacollector.dll";
+            public const string CoverletDataCollectorCodebase = "coverlet.collector.dll";
```
Comment (singhsarab) [inline src/vstest.console/Processors/CollectArgumentProcessor.cs]: What is the name of the out proc one ?

Comment (vagisha-nidhi) [reply src/vstest.console/Processors/CollectArgumentProcessor.cs]: There is only one dll coverlet.collector. The inproc datacollector is inside this in a separate folder. 
Link: [https://github.com/tonerdo/coverlet/tree/master/src/coverlet.collector](https://github.com/tonerdo/coverlet/tree/master/src/coverlet.collector)

**Review by cltshivash — APPROVED** [review]

File: src/vstest.console/Processors/CollectArgumentProcessor.cs (hunk 1 above)
Comment (cltshivash) [reply src/vstest.console/Processors/CollectArgumentProcessor.cs]: The publicKeyToken can be removed if null

Comment (MarcoRossignoli) [reply src/vstest.console/Processors/CollectArgumentProcessor.cs]: Guys if I clone repo and do some update on public key...is there a guide that explain how to run my local "dotnet test" driver?
If so I could test on my local if there are issue with collector load and do a PR directly to this repo.

**Pull request merged**